sympy/stats/crv.py

```diff
@@ -145,6 +150,164 @@ def __call__(self, *args):
+            a=float(dist.alpha), size=size) + 1) * float(dist.xm),
+        'UniformDistribution': lambda dist, size: numpy.random.uniform(
+            low=float(dist.left), high=float(dist.right), size=size)
+    }
+
+    python_rv_map = {
+        'BetaDistribution': lambda dist:
+            random.betavariate(float(dist.alpha), float(dist.beta)),
+        'ExponentialDistribution': lambda dist:
+            random.expovariate(float(dist.rate)),
+        'GammaDistribution': lambda dist:
+            random.gammavariate(float(dist.k), float(dist.theta)),
+        'LogNormalDistribution': lambda dist:
+            random.lognormvariate(float(dist.mean), float(dist.std)),
+        'NormalDistribution': lambda dist:
+            random.normalvariate(float(dist.mean), float(dist.std)),
+        'ParetoDistribution': lambda dist:
+            random.paretovariate(float(dist.alpha)),
+        'UniformDistribution': lambda dist:
+            random.uniform(float(dist.left), float(dist.right)),
+        'WeibullDistribution': lambda dist:
+            random.weibullvariate(float(dist.alpha), float(dist.beta)),
+        'ContinuousDistributionHandmade': lambda dist:
+            dist._inverse_cdf_expression()(random.uniform(0, 1))
+        }
+
+    pymc3_rv_map = {
+        'BetaDistribution': lambda dist:
+            pymc3.Beta('X', alpha=float(dist.alpha), beta=float(dist.beta)),
+        'CauchyDistribution': lambda dist:
+            pymc3.Cauchy('X', alpha=float(dist.x0), beta=float(dist.gamma)),
+        'ChiSquaredDistribution': lambda dist:
+            pymc3.ChiSquared('X', nu=float(dist.k)),
+        'ExponentialDistribution': lambda dist:
+            pymc3.Exponential('X', lam=float(dist.rate)),
+        'GammaDistribution': lambda dist:
+            pymc3.Gamma('X', alpha=float(dist.k), beta=1/float(dist.theta)),
+        'LogNormalDistribution': lambda dist:
+            pymc3.Lognormal('X', mu=float(dist.mean), sigma=float(dist.std)),
+        'NormalDistribution': lambda dist:
+            pymc3.Normal('X', float(dist.mean), float(dist.std)),
+        'GaussianInverseDistribution': lambda dist:
+            pymc3.Wald('X', mu=float(dist.mean), lam=float(dist.shape)),
+        'ParetoDistribution': lambda dist:
+            pymc3.Pareto('X', alpha=float(dist.alpha), m=float(dist.xm)),
+        'UniformDistribution': lambda dist:
+            pymc3.Uniform('X', lower=float(dist.left), upper=float(dist.right))
+    }
+
+    @classmethod
+    def _sample_scipy(cls, dist, size):
+        """Sample from SciPy."""
+
+        dist_list = cls.scipy_rv_map.keys()
+
+        if dist.__class__.__name__ not in dist_list:
+            return None
+
+        if dist.__class__.__name__ == 'ContinuousDistributionHandmade':
+            from scipy.stats import rv_continuous
+            handmade_pdf = dist.pdf
+            class scipy_pdf(rv_continuous):
+                def _pdf(self, z):
+                    return handmade_pdf(z)
```

Comment:
How is it supposed to operate?
I think that sympy functions should be lambdified before passing to scipy.